src/osp/Active/SysWire.h

```diff
@@ -287,11 +287,11 @@ class WireOutput : private LinkedList<WireInput>
 //-----------------------------------------------------------------------------
 
 
-class SysWire : public IDynamicSystem
+class SysWire
 {
 public:
 
-    static const std::string smc_name;
+    static inline std::string smc_name = "Wire";
```

Comment:
The other systems seem to have lost this member, should this one have stayed?

For the rest, is the `smc_name` concept being removed entirely?